packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/CompactOnLaunchTests.kt

```diff
@@ -69,10 +69,11 @@ class CompactOnLaunchTests {
     fun defaultCallback_boundaries() {
         val callback = Realm.DEFAULT_COMPACT_ON_LAUNCH_CALLBACK
         assertFalse(callback.shouldCompact(50 * 1024 * 1024, 40 * 1024 * 1024))
-        assertFalse(callback.shouldCompact(50 * 1024 * 1024 + 8, 25 * 1024 * 1024))
-        assertFalse(callback.shouldCompact(50 * 1024 * 1024 + 8, 25 * 1024 * 1024 + 3))
+        assertTrue(callback.shouldCompact(50 * 1024 * 1024 + 8, 25 * 1024 * 1024))
```

Comment:
I think that this test readability could be improved, and would have help catching the issue earlier:
Add parameters names
Use some kind of constants/functions, like the duration framework

```suggestion
assertTrue(callback.shouldCompact(totalBytes = 50.MB + 1, usedBytes = 25.MB))
```

Reply:
Totally agree on the readability of the test. Didn't want to change the test more than strictly necessary. Thanks!